s_tui/Sources/TemperatureSource.py

```diff
@@ -51,6 +51,8 @@ def __init__(self, custom_temp=None, temp_thresh=None):
                 self.temp_thresh = int(temp_thresh)
```

Comment:
While we are at it, its a good idea to address the case of `ValueError` for this conversion, and set to THRESHOLD_TEMP if it fails

Reply:
Made a new commit that reflects all this.